pull_request_template.md

```diff
@@ -30,7 +32,6 @@ If you have any questions or problems, please give a comment in the PR.
 - [ ] Code comments are correct (logically and spelling)
 - [ ] Required documentation changes are present and part of the PR
 - [ ] `bareos-check-sources --since-merge` does not report any problems
-- [ ] `git status` should not report modifications in the source tree after building and testing
```

Comment:
Why did we remove that line?

Reply:
@pstorz said, that this have not happened for a long time and can therefore be removed.

Reply:
I would have keep it, especially for external contribution, but we can try without and see.